fusor-ember-cli/app/components/cancel-back-next.js

```diff
@@ -11,16 +11,17 @@ export default Ember.Component.extend({
 
     saveAndCancelDeployment() {
       this.get('targetObject').send('saveAndCancelDeployment');
-      this.set('openModal', false);
+      return this.set('openModal', false);
```

Comment:
I think seeing a "return" from an action is usually an attempt to bubble an action up the DOM (return true) or not (return false).  It's not clear to me what this.set(...) will return, but we could get some weird behavior depending on it changing.  Are we intending the actions to bubble?  If so we should probably return true, if not, we should either not return (returns undefined) or return false.